src/frontend/classic/components/MediaSelect.svelte

```diff
@@ -49,13 +50,16 @@
 
     let mediaNameFilter = "";
     $: {
-        filteredmedias = medias.filter((item) => {
-            return (
-                item?.Parent?.Title?.toLowerCase().indexOf(
-                    mediaNameFilter.toLowerCase()
-                ) !== -1
-            );
-        });
+        if (mediaNameFilter === "") filteredmedias = medias;
+        else {
+          const fuse = new Fuse(medias, {
```

Comment:
medias being a fixed list. It should be initialized outside the reactive statement when medias is loaded

Reply:
Thank you for your kind suggestion, the proposed change has been applied at https://github.com/manga-download/haruneko/pull/37/commits/71a38ab46e4e33134c5ccf8766df3b3d90bfe9d1